data/darktableconfig.xml.in

```diff
@@ -2237,16 +2237,16 @@
     <longdescription>process the image in darkroom mode with a small border. set to 0 if you don't want any border.</longdescription>
   </dtconfig>
   <dtconfig>
-    <name>darkroom/ui/iso12464_border</name>
+    <name>darkroom/ui/color_assessment_border</name>
```

Comment:
@TurboGit how do we make sure existing darktablerc configs are updated with the new config param names?

Reply:
Understood, makes sense :+1: